# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/tests/css/css-contain/content-visibility/content-visibility-070.html
# Commit: c428c3ad7725
# Full Hash: c428c3ad7725593f33b4ea2da9851be31be0c1df
# Author: Martin Robinson <mrobinson@igalia.com>
# Date: 2022-11-28 15:41:50
# Regressor Bug: 1791759
# File Overlap Count: 1
# Description:
#   Bug 1791759 - Add support for `content-visibility: auto` r=emilio
#   
#   This change adds support for `content-visibilty: auto` as well as
#   showing and hiding content based on the relevancy of the content as
#   defined in the specification. Changes to relevancy are handled by
# ==============================================================================

diff -r ce553434720c -r c428c3ad7725 testing/web-platform/tests/css/css-contain/content-visibility/content-visibility-070.html
--- a/testing/web-platform/tests/css/css-contain/content-visibility/content-visibility-070.html	Mon Nov 28 09:45:39 2022 +0000
+++ b/testing/web-platform/tests/css/css-contain/content-visibility/content-visibility-070.html	Mon Nov 28 09:53:06 2022 +0000
@@ -59,8 +59,8 @@
   function step2() {
     const r = container.getBoundingClientRect();
     t.step(() => {
-      assert_equals(r.y, 3000, "step3 offset");
-      assert_equals(r.height, 10, "step3 height");
+      assert_equals(r.y, 3000, "step2 offset");
+      assert_equals(r.height, 10, "step2 height");
     });
     document.scrollingElement.scrollTop = 3000;
     requestAnimationFrame(step3);
@@ -69,8 +69,8 @@
   function step3() {
     const r = container.getBoundingClientRect();
     t.step(() => {
-      assert_less_than(r.y, 3000, "step2 offset");
-      assert_equals(r.height, 10, "step2 height");
+      assert_less_than(r.y, 3000, "step3 offset");
+      assert_equals(r.height, 10, "step3 height");
     });
     document.scrollingElement.scrollTop = 0;
     requestAnimationFrame(step4);
@@ -88,8 +88,8 @@
   function step5() {
     const r = container.getBoundingClientRect();
     t.step(() => {
-      assert_equals(r.y, 3000, "step4 offset");
-      assert_equals(r.height, 10, "step4 height");
+      assert_equals(r.y, 3000, "step5 offset");
+      assert_equals(r.height, 10, "step5 height");
     });
 
     selection.removeAllRanges();